firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionNotificationsManager.java

```diff
@@ -65,24 +65,24 @@ void updateNotification(long totalBytes, long downloadedBytes, UpdateStatus stat
   }
 
   int getNotificationContentTitleId(UpdateStatus status) {
-    if (isErrorState(status)) {
-      return R.string.download_failed;
-    } else if (status.equals(UpdateStatus.DOWNLOADED)) {
-      return R.string.download_completed;
-    } else {
-      return R.string.downloading_app_update;
+    switch (status) {
+      case DOWNLOAD_FAILED:
+        return R.string.download_failed;
+      case INSTALL_FAILED:
+        return R.string.install_failed;
+      case INSTALL_CANCELED:
+        return R.string.install_canceled;
+      case NEW_RELEASE_CHECK_FAILED:
+        return R.string.new_release_check_failed;
+      case UPDATE_CANCELED:
+        return R.string.update_canceled;
+      case DOWNLOADED:
+        return R.string.download_completed;
+      default:
+        return R.string.downloading_app_update;
```

Comment:
It's a little strange to default to "Downloading..." For example why is "Downloading..." the right message when the status is PENDING? https://github.com/firebase/firebase-android-sdk/blob/3072bbcdb5416b0c1accda91a9cd9f24cde6063e/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java#L182

I think ideally the callers to `updateNotification()` would just pass in a string resource ID directly, and then we wouldn't need to do this Enum -> String translation here at all.

Reply:
Added in the case for pending to return that it is pending. That should cover all the cases that we use in ApkUpdater. My sense is that will only be in the notification for a small time (even smaller than downloading but that can take more time depending on file size and internet speed). With the next change there shouldn't be a default regardless. 

Why is that ideal? Made the change to pass both, because postUpdateProgress needs the status for the updateProgress builder.